templates/ai/install.html

```diff
@@ -0,0 +1,114 @@
+{% extends "ai/base_ai.html" %}
+{% block title %}Install Kubeflow yourself{% endblock %}
+{% block meta_description %}A step-by-step installation guide to Ubuntu OpenStack with Kubernetes and Kubeflow on bare metal servers.{% endblock %}
+
+{% block content %}
+<section class="p-strip--light is-deep is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h1>Install Kubeflow yourself</h1>
+      <p>Canonical continues to innovate and develop best practices for supporting developers and operators.</p>
+    </div>
+    <div class="col-4 prefix-1 u-align--center u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}dba11aee-kubernetes.svg?w=275" alt="" width="275" />
```

Comment:
From an SEO standpoint, it would be good to have `alt` attributes relevant to the page or to the image when the image is descriptive. In this case, I'm suggesting: `alt="How to install Kubeflow"`

Reply:
The alt tags are for assistive technology. For example screen readers. If an image has no relevance to the content of the page, it is recommended to leave the alt blank so the screen reader can skip it. I am not aware of any search engines that index based on image alt attributes?